rasa/core/training/interactive.py

```diff
@@ -816,10 +818,15 @@ def _write_stories_to_file(
     else:
         append_write = "w"  # make a new file if not
 
+    with open(
+        export_story_path, "r", encoding=rasa.shared.utils.io.DEFAULT_ENCODING
```

Comment:
Why you are using "r"? Maybe do something like append_write variable in line 819 is better.

Reply:
In this code we are only reading the file to find the "interactive_story" string. There's no need to use append_write.